test/integration/ads_integration_test.cc

```diff
@@ -1546,4 +1546,42 @@ TEST_P(AdsClusterV2Test, DEPRECATED_FEATURE_TEST(TypeUrlAnnotationRegression)) {
   test_server_->waitForCounterGe("cluster_manager.cds.update_rejected", 1);
 }
 
+class AdsSetNodeAlwaysIntegrationTest : public AdsIntegrationTest {
+public:
+  AdsSetNodeAlwaysIntegrationTest() = default;
```

Comment:
nit: this is not necessary, just remove this line

Reply:
thank you!